src/saleor_mcp/telemetry/trace.py

```diff
@@ -0,0 +1,38 @@
+from typing import Sequence
+
+from opentelemetry import trace
+from opentelemetry.context import Context as OTContext
+from opentelemetry.trace import SpanKind, Link
+from opentelemetry.util.types import Attributes
+
+tracer = trace.get_tracer("saleor-mcp")
+
+
+def start_as_current_span(
```

Comment:
Do we need this wrapper for anything other than basic `tracer.start_as_current_span`?